src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTypeCheck.java

```diff
@@ -569,10 +586,10 @@ private void visitParameterDef(DetailAST parameterDef) {
     }
 
     /**
-     * Checks type of given variable.
-     *
-     * @param variableDef variable to check.
-     */
+         * Checks type of given variable.
+         *
+         * @param variableDef variable to check.
+         */
```

Comment:
Please revert this.

Reply:
Done Changes

Reply:
Not done

Reply:
Sir, It is done and I am just pushing my code in a minute.

Reply:
resolved